project.clj

```diff
@@ -19,7 +19,7 @@
                  [reagent "0.5.1"]
                  [markdown-clj "0.9.89"]
                  ;; Not sure if this should just be a dev dep; It's kinda nice
-                 [data-frisk-reagent "0.2.3"]
+                 [data-frisk-reagent "0.2.4"]
```

Comment:
@bamarco Weird... Didn't realize we were on `.3` Hopefully we'll be able to bump to `.5` tonight